modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -61,12 +61,23 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		CookiesConstants.NAME_ID;
 		CookiesConstants.NAME_USER_UUID;
 
+		AssetListEntryCreateDateComparator.getInstance(false);
+		AssetTagCountComparator.getInstance(false);
+
+		StructureStructureKeyComparator.getInstance(true);
+		StructureStructureKeyComparator.getInstance(false);
```

Comment:
These calls should be sorted.

```
		StructureStructureKeyComparator.getInstance(false);
		StructureStructureKeyComparator.getInstance(true);
```

Reply:
And it should come after the `PortletDisplayTemplateConstants` and `ServiceTrackerList` cases.